cypress/integration/entitybrowser_spec.js

```diff
@@ -5,28 +5,49 @@ describe('Entity Browser', function() {
     cy.login()
   })
 
-  beforeEach(() => {
-    cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}`)
-  })
-
   describe('ListView', () => {
+    beforeEach(() => {
+      cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}`)
+    })
+
     it('should load and display essential parts of the list view', function() {
+      cy.contains('Daten werden geladen...', {timeout: 15000})
       cy.get('[title="Vorname"]')
       cy.get('[type="submit"]')
+      cy.contains('Einträgen')
       cy.contains('Aktionen auf Alle')
-      cy.get('[data-list-cell]').should('have.length.above', 1)
+      cy.get('[data-cy=list-cell]', {timeout: 6000})
+        .should('have.length.above', 10)
+    })
+
+    describe('SearchForm', () => {
+      it('should display extended search form', function() {
+        cy.get('[data-cy=form-field]')
+          .should('have.length', 1)
+        cy.get('[data-cy=extend-search-button]')
+          .click()
+        cy.get('[data-cy=form-field]')
+          .should('have.length.above', 10)
```

Comment:
I would choose a slightly different approach when it comes to the selection of assertion criteria that make sense.

For instance, `10` is a magic arbitrary number. We actually don't care if there are more than 10 fields or if there are only 9. The only thing we really care about is that there is more than 1 field. So, it would be perfectly fine to simply assert that there is more than 1. This would also make the test more stable and leave less room for speculation (someone who stumbles over this test for some reason cannot know if that number `10` actually is important or not).